DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -600,5 +609,50 @@ public void ReactivateCentre(int centreId)
                 new { centreId }
             );
         }
+        
+        public CentreSummaryForRoleLimits GetRoleLimitsForCentre(int centreId)
+        {
+            return connection.QueryFirstOrDefault<CentreSummaryForRoleLimits>(
+                @"SELECT CentreId,
+                        RoleLimitCMSAdministrators,

```

Comment:
We should tweak the data service calls to reference the existing fields too.

Reply:
Done.